Source/common/SNTConfigurator.m

```diff
@@ -549,6 +560,21 @@ - (void)setRemountUSBMode:(NSArray<NSString *> *)args {
   return args;
 }
 
+- (NSDictionary<NSString *, SNTRule *> *)staticRules {
+  NSArray *currentRules = self.configState[kStaticRules];
+  if (currentRules.hash != self.cachedStaticRulesHash) {
+    NSMutableDictionary *rules = [NSMutableDictionary dictionaryWithCapacity:currentRules.count];
+    for (id rule in currentRules) {
+      if (![rule isKindOfClass:[NSDictionary class]]) return self.cachedStaticRules;
+      SNTRule *r = [[SNTRule alloc] initWithDictionary:rule];
```

Comment:
I think we should log a warning here if the dictionary is malformed, so admins can figure out if they've typo-ed a config.

Reply:
We can't log inside SNTConfigurator as that would create a circular dependency between SNTConfigurator and SNTLogging.

Reply:
Right, we're using `SNTConfigurator` to set the logging level. Probably a good idea to split this up in the future so that `SNTConfigurator` can set the log level and log without the circular dependency, but outside the scope of this PR.